Cabal/Distribution/Utils/ShortText/Internal.hs

```diff
@@ -0,0 +1,107 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE DeriveDataTypeable #-}
+{-# LANGUAGE DeriveGeneric #-}
+
+module Distribution.Utils.ShortText.Internal
```

Comment:
Please use the WARNING scheme we use for the other `.Internal`-modules, see `Distribution.Compat.Prelude.Internal`